pyproject.toml

```diff
@@ -44,6 +45,7 @@ dependencies = [
     "scipy",
     "pyqtorch==1.0.6",
     "matplotlib",
+    "Arpeggio==2.0.2",
```

Comment:
does this repo require some binaries or can it just be pip-installed? would be nice to confirm that on the cluster so we can be sure it can be installed properly in the containers

Reply:
@dominikandreasseitz just confirming: there's not binary dependency; it can be pip-installed directly.

The only feature it contains external dependencies (optional), and that will not be used on cluster,  is the graphviz for the dot files.